### PR TITLE
CI build/publish pipeline

### PR DESCRIPTION
**Pull request by beorosz**
Build and publish SigSpec.* Nuget packages. Not sure why the changes from the previous PR (unregisterCallbacks) are reintroduced - I must have executed the merge in a wrong way :(
The build/publish is using service connections from https://dev.azure.com/belaorosz/SigSpec project.

### CODE REVIEW

**Comment by beorosz** [comment]
The current package push to Nuget.org is using my API key, configured in the Azure Devops project mentioned above.. This could be replaced if needed.

**Comment by RicoSuter** [comment]
Cant we just use project properties instead of the nuspec files?

**Comment by RicoSuter** [comment]
Is this pr ready?

**Comment by beorosz** [comment]
Sorry - yes, it is.

File: src/SigSpec.CodeGeneration.TypeScript/SigSpec.CodeGeneration.TypeScript.csproj [hunk 1]
```diff
@@ -1,9 +1,22 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <!-- Nuget-related properties -->
+  <PropertyGroup>
```
Comment (RicoSuter) [inline src/SigSpec.CodeGeneration.TypeScript/SigSpec.CodeGeneration.TypeScript.csproj]: Is this group needed?

Comment (beorosz) [reply src/SigSpec.CodeGeneration.TypeScript/SigSpec.CodeGeneration.TypeScript.csproj]: These props control the Nuget package creation. I wanted to put them into a separate group, so they are easier to find and modify when needed.

**Comment by RicoSuter** [comment]
Sorry for the late reply. Thx for the pr.

**Pull request merged**